pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -206,12 +206,13 @@ type TidbClusterSpec struct {
 
 // TidbClusterStatus represents the current status of a tidb cluster.
 type TidbClusterStatus struct {
-	ClusterID string        `json:"clusterID,omitempty"`
-	PD        PDStatus      `json:"pd,omitempty"`
-	TiKV      TiKVStatus    `json:"tikv,omitempty"`
-	TiDB      TiDBStatus    `json:"tidb,omitempty"`
-	Pump      PumpStatus    `josn:"pump,omitempty"`
-	TiFlash   TiFlashStatus `json:"tiflash,omitempty"`
+	ClusterID string          `json:"clusterID,omitempty"`
+	PD        PDStatus        `json:"pd,omitempty"`
+	TiKV      TiKVStatus      `json:"tikv,omitempty"`
+	TiDB      TiDBStatus      `json:"tidb,omitempty"`
+	Pump      PumpStatus      `josn:"pump,omitempty"`
+	TiFlash   TiFlashStatus   `json:"tiflash,omitempty"`
+	Monitor   *TidbMonitorRef `json:"monitor,omitempty"`
```

Comment:
Where is this reference used?
How does this PR fix #2331?

Reply:
Here is the first step to realize #2331